src/components/engagement/engagement.gel.repository.ts

```diff
@@ -63,19 +63,51 @@ const languageHydrate = e.shape(e.LanguageEngagement, (le) => ({
   ...baseHydrate(le),
   __typename: le.__type__.name,
   ...languageExtraHydrate,
+  label: e.select({
+    project: le.project.name,
+    language: le.language.name,
+    intern: e.cast(e.str, e.set()),
```

Comment:
Why does this `intern` property have to be up here in the `languageHydrate`?

Reply:
Because all engagements have this property, so we need to produce the null here for intern engagements.
https://github.com/SeedCompany/cord-api-v3/blob/373939981aad28b1625be54f0266a37a974c0c18/src/components/engagement/dto/engagement.dto.ts#L87

Reply:
You mean produce the null for language engagements?

Reply:
My b, correct